collector/runit.go

```diff
@@ -18,10 +18,10 @@ package collector
 
 import (
 	"github.com/alecthomas/kingpin/v2"
+	"github.com/discordianfish/go-runit/runit"
```

Comment:
I'd prefer to fork this to `prometheus-community`, what do you think about that?

Reply:
done